src/view/com/profile/ProfileSubpageHeader.tsx

```diff
@@ -103,11 +104,7 @@ export function ProfileSubpageHeader({
                 style={[styles.backIcon, pal.text]}
               />
             ) : (
-              <FontAwesomeIcon
-                size={18}
-                icon="bars"
-                style={[styles.backIcon, pal.textLight]}
-              />
+              <Menu size="lg" style={[{marginTop: 4}, pal.textLight]} />
```

Comment:
Replaced the burger in all locations to use new icon set + made sure it aligns properly on each screen, it didn't before